docker/jenkins/build.sh

```diff
@@ -23,10 +23,10 @@ valid_images=(
   py2-android-ubuntu16.04
 
   # Builds for Anaconda
-  py2-conda-ubuntu16.04
-  py2-conda-cuda9.0-cudnn7-ubuntu16.04
-  py3-conda-ubuntu16.04
-  py3-conda-cuda9.0-cudnn7-ubuntu16.04
+  conda2-ubuntu16.04
+  conda2-cuda9.0-cudnn7-ubuntu16.04
+  conda3-ubuntu16.04
+  conda3-cuda9.0-cudnn7-ubuntu16.04
```

Comment:
Are these actually used anywhere? I think we can just delete them

Reply:
They aren't, but can serve as documentation.